vita3k/modules/SceVshBridge/SceVshBridge.cpp

```diff
@@ -314,7 +314,8 @@ EXPORT(int, _vshSysconCtrlManualChargeMode) {
 }
 
 EXPORT(int, _vshSysconGetHardwareInfo) {
```

Comment:
Looking at the disassembly for _vshSysconGetHardwareInfo, it has signature `int _vshSysconGetHardwareInfo(char info[4])` and returns 0 in case of success (the hardware info is stored in info). Also I'm not really sure if its worth implementing this function if the hardware info returned is completely opaque.

Reply:
`GetHardwareInfo` didn't had any argument in this instance. Although this is the kernel call.
```
      uVar2 = SceSysconForDriver::GetHardwareInfo_CBD6D8BC();
      if (((uVar2 & 0xff0000) == 0x300000) || ((uVar2 & 0xff0000) == 0x310000)) {
        PixelDensity1 = 22.0;
        FLOAT_810073e4 = 22.0;
        PixelDensity2 = 22.0;
        FLOAT_810074d8 = 22.0;
      }
```

I don't have any necessity to implement it at the moment. So I'm going to just revert it in the meantime.

Reply:
Ah ok, the kernel space call (`ksceSysconGetHardwareInfo`) directly returns the hardware info while the user one `vshSysconGetHardwareInfo` calls the former but returns an error code and stores the hardware info in an address given in parameters.